app/containers/Forms/Form.tsx

```diff
@@ -98,6 +98,31 @@ export const FormComponent: React.FunctionComponent<Props> = ({
       return result?.requiresEmissionAllocation === true;
     });
 
+  const customValidation = (formData, errors) => {
+    let hasFalseRequiresEmissionAllocation = false;
+    let nonEnergyProductCount = 0;
+    const productsInConflict = [];
+    if (formData[0]?.productRowId) {
+      formData.forEach((product, index: number) => {
+        if (product.requiresEmissionAllocation === false)
+          hasFalseRequiresEmissionAllocation = true;
+        if (product.productRowId > 7) {
```

Comment:
This needs a TechDebt card to add an `is_energy_product` boolean and a comment in `schema/data/prod/energy_product.sql`, indicating that this line needs to be changed if any energy product is added/removed.